comment/templatetags/comment_tags.py

```diff
@@ -139,9 +140,17 @@ def is_user_blocked(comment):
     return BlockedUser.objects.is_user_blocked(user_id, comment.email)
 
 
+# TODO: remove in v3.0.0
 @register.simple_tag(name='include_static')
 def include_static():
     """ This function shall be deprecated """
+    warnings.warn(
+        (
+            'The tag `include_static` has been deprecated. Static files are now rendered implicitly.'
+            'You can remove this from your django template. This tag will be removed in a v3.0.0.'
```

Comment:
Can you open an issue for this change and assign _3.0.0_ milestone?

Reply:
#216 aims to track this.